meinberlin/apps/kiezradar/migrations/0001_initial.py

```diff
@@ -49,9 +49,10 @@ class Migration(migrations.Migration):
                 ("name", models.CharField(max_length=125)),
                 (
                     "point",
-                    adhocracy4.maps.fields.PointField(
+                    django.contrib.gis.db.models.fields.PointField(
```

Comment:
I've just reset the migrations for `kiezradar` again so we don't have to have a another delete and create

Reply:
instead of resetting the migration I just adde a new one for now, so we don't all have to scrap our dbs agian